cluster/search/load_data.py

```diff
@@ -90,12 +107,12 @@ def add_indico(documents):
         analysis["keywords"] = EXECUTOR.submit(indicoio.keywords, [doc.get("text") for doc in documents], version=1)
         analysis["ner"] = EXECUTOR.submit(indicoio.named_entities, [doc.get("text") for doc in documents], version=2)
 
-        analysis["title_sentiment_hq"] = analysis["title_sentiment_hq"].result()
-        analysis["title_keywords"] = analysis["title_keywords"].result()
-        analysis["title_ner"] = analysis["title_ner"].result()
-        analysis["sentiment_hq"] = analysis["sentiment_hq"].result()
-        analysis["keywords"] = analysis["keywords"].result()
-        analysis["ner"] = analysis["ner"].result()
+        analysis["title_sentiment_hq"] = try_except_result(analysis["title_sentiment_hq"], -1, individual=len(documents) == 1)
+        analysis["title_keywords"] = try_except_result(analysis["title_keywords"], [], individual=len(documents) == 1)
```

Comment:
You could set the `individual` argument once instead of inlining the `len(documents) == 1` call a bunch of times.